Adding new directive CLEANUP to test-suite (#2610)
iovisor/bpftrace: tests/README.md

  will be terminated after testcase is over.
* `AFTER`: Run the command in a shell after running bpftrace. The command will
  be terminated after the testcase is over.

> * `CLEANUP`: Run the command in the shell after BEFORE/RUN/AFTER commands have
>   terminated. This holds any cleanup command to free resources after test has

Reviewer: ```suggestion
* `CLEANUP`: Run the command in a shell after testcase is over. This holds any
 cleanup command to free resources after test has
```
